llvm/test/CodeGen/Hexagon/loop-idiom/lcssa.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: opt -S -hexagon-loop-idiom < %s | opt -S -passes='loop(loop-deletion),gvn'
+; RUN: opt -mtriple hexagon-- -S -passes='loop(hexagon-loop-idiom,loop-deletion),gvn'
 ; REQUIRES: asserts
```

Comment:
this REQUIRES doesn't make sense, we should run it under all configurations...

same for the other tests here

Reply:
Yes, this test should be run under all configurations. REQUIRES should be removed.

Reply:
agreed, but should be done in a separate PR

Reply:
Sure, I'll push a patch to remove REQUIRES.